### ISSUE TITLE
Update Permission for Android 13

### ISSUE THREAD

**Issue by DucLongDTH** [chavesgu/images_picker#92]
Hi , can you update lib for android 13 have permission read and write data
![Screenshot_1663666521](https://user-images.githubusercontent.com/61280727/191223601-015b22c7-ec04-4905-905e-8ffee13ee4e5.png)
I also allow
![Screenshot_1663666582](https://user-images.githubusercontent.com/61280727/191224674-f9796fd3-fb77-425e-8dc3-a3e3a1c937f5.png)
ed all the permissions but when back to my app there is no change !

**Comment by Starkm12** [comment]
Any update on this?

**Comment by An5675** [comment]
Hi, the problem is not strictly related to the package (images_picker),
but it is a modification of the permissions Android 13

source 1:
https://developer.android.com/about/versions/13/behavior-changes-13

source 2:
https://issuetracker.google.com/issues/228868661

I solved it by adding these permissions to
.. \ android \ app \ src \ main \ AndroidManifest.xml

<uses-permission android: name = "android.permission.READ_MEDIA_IMAGES" />
<uses-permission android: name = "android.permission.READ_MEDIA_VIDEO" />


and modifying
.. \ android \ app \ build.gradle


targetSdkVersion 32 instead of -> targetSdkVersion 33
compileSdkVersion 33

**Comment by DucLongDTH** [comment]
> Any update on this?

yah, not yet, i fixed the native library but i can't implement into my fork. The problem appears in a native library when it is too old

**Comment by DucLongDTH** [comment]
> Hi, the problem is not strictly related to the package (images_picker), but it is a modification of the permissions Android 13
> 
> source 1: https://developer.android.com/about/versions/13/behavior-changes-13
> 
> source 2: https://issuetracker.google.com/issues/228868661
> 
> I solved it by adding these permissions to .. \ android \ app \ src \ main \ AndroidManifest.xml
> 
> and modifying .. \ android \ app \ build.gradle
> 
> targetSdkVersion 32 instead of -> targetSdkVersion 33 compileSdkVersion 33

yes, i already do it but btw I used another library and i can set to targetSdkVersion 33 compileSdkVersion 33 ^^

**Comment by leonardobenedeti** [comment]
@DucLongDTH it is possible for you submit a pull request or publish this changes to another pub ?

**Comment by DucLongDTH** [comment]
> @DucLongDTH it is possible for you submit a pull request or publish this changes to another pub ?

@leonardobenedeti i think this pub libary not updated or updated in this lib: https://pub.dev/packages/image_pickers. this support android 13

**Comment by lamlengend98** [comment]
@DucLongDTH how can I fix this in the native?

**Comment by LongND-NTQ** [comment]
> @DucLongDTH how can I fix this in the native?

Cannot update in native code, you can use package image_pickers, the update version of this package

**Comment by roshansrashtasoft** [comment]
> yes, i already do it but btw I used another library and i can set to targetSdkVersion 33 compileSdkVersion 33 ^^

@An5675 I am trying but it is not working my project ?

Any other solution ?

**Comment by roshansrashtasoft** [comment]
> Hi, the problem is not strictly related to the package (images_picker), but it is a modification of the permissions Android 13
> 
> source 1: https://developer.android.com/about/versions/13/behavior-changes-13
> 
> source 2: https://issuetracker.google.com/issues/228868661
> 
> I solved it by adding these permissions to .. \ android \ app \ src \ main \ AndroidManifest.xml
> 
> and modifying .. \ android \ app \ build.gradle
> 
> targetSdkVersion 32 instead of -> targetSdkVersion 33 compileSdkVersion 33

@An5675  I am trying but it is not working my project ?

Any other solution ?
